cluster/vagrant/util.sh

```diff
@@ -288,6 +303,11 @@ function restart-kube-proxy {
   ssh-to-node "$1" "sudo systemctl restart kube-proxy"
 }
 
+# Restart the apiserver
+function restart-apiserver {
```

Comment:
This seems like a fine provider function. Any reason not to splat it across the different cloud providers rather than special casing it in `services.sh`? (Other than expediency.)